app/controllers/api/v1/users_controller.rb

```diff
@@ -19,5 +19,7 @@ def show
         nickname: @steam.nickname
       }
     }
+  rescue ActiveRecord::RecordNotFound
+    raise ActionController::RoutingError.new('User Not Found')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.